Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/tracing/details/TracingDetailsStateProvider.kt

```diff
@@ -38,20 +39,22 @@ class TracingDetailsStateProvider @Inject constructor(
         val (latestCalc, latestSuccessfulCalc) = riskLevelResults.tryLatestResultsWithDefaults()
 
         val isAdditionalInformationVisible = riskDetailPresenter.isAdditionalInfoVisible(
-            latestCalc.riskLevel.raw, latestCalc.matchedKeyCount
+            latestCalc.riskState, latestCalc.matchedKeyCount
         )
         val isInformationBodyNoticeVisible = riskDetailPresenter.isInformationBodyNoticeVisible(
-            latestCalc.riskLevel.raw
+            latestCalc.riskState
         )
 
+        val isRestartButtonEnabled = !isBackgroundJobEnabled || latestCalc.riskState == RiskState.CALCULATION_FAILED
```

Comment:
Same here isBackgroundJosDisabled instead of negation

Reply:
Then we would have to negate `isAutoModeEnabled` and negate it again in other places.

Reply:
I thought about having both, like val isBackgroundJobDisabled = !isBackgroundJobEnabled